.github/dependabot.yml

```diff
@@ -0,0 +1,17 @@
+# To get started with Dependabot version updates, you'll need to specify which
+# package ecosystems to update and where the package manifests are located.
+# Please see the documentation for all configuration options:
+# https://docs.github.com/github/administering-a-repository/configuration-options-for-dependency-updates
+
+version: 2
+updates:
+  - package-ecosystem: "github-actions" # See documentation for possible values
+    directory: "/" # Location of package manifests
+    schedule:
+      interval: "daily"
+      
+   - package-ecosystem: "docker" # See documentation for possible values
+    directory: "/" # Location of package manifests
+    schedule:
+      interval: "daily"
+      
```

Comment:
This file is invalid YAML, since the indentation is wrong, see:

    Error : bad indentation of a sequence entry at line 13, column 4:
           - package-ecosystem: "docker" #  ... 
           ^
    Line : undefined  undefined

Reply:
Good catch. Indentation fixed.